api_mt/Trados.md

```diff
@@ -0,0 +1,121 @@
+## 2. How to set up Trados Studio 2021™ plugins
+### 2.1 Amazon Translate (AWS)
+#### 2.1.1 How to obtain an API key
+Refer to [Point 4.1](#41-amazon-translate-aws) for details.
+
+#### 2.1.2 How to set up the Trados Studio 2021™ plugin for Amazon Translate (AWS)
+ - Open your desktop version of Trados Studio 2021™.
```

Comment:
In my opinion, imperatives like this should not end with a `.`

What do you and @cefoo think?  What do the Microsoft style guide and Chicago Manual of Style say about this?

Reply:
The Chicago Manual of Style says the following:

> If the list is unordered, and unless the items consist of complete sentences, each item carries no end punctuation and each can usually begin lowercase (except for proper nouns).
> If the list is ordered, closing punctuation is used only if items consist of complete sentences.

Reply:
Generally speaking I followed Google guidelines when writing the manual. As far as I remember such sentences are to be ended with a  fullstop. Plus since "So, according to the Chicago Manual of Style, these list items should end with a .." I leave the fullstops. Ok?

Reply:
We have decided to have Machine Translate in en_UK and follow the [European Commission English Style Guide](https://ec.europa.eu/info/sites/default/files/styleguide_english_dgt_en.pdf).

According to the European Commission, if list items contain more than one sentence, then they should start with uppercase and end with a period (section 11.5). I think that is the case here.